File: POOPlatzi/PHP/uberPool.php
<?php
require_once("Car.php");
class uberPool extends Car {

   public $marca;
   public $modelo;

   public function __construct($licencia,$conductor,$marca,$modelo){

   parent::__construct($licencia,$conductor);
   $this->marca=$marca;
   $this->modelo=$modelo;

   }
}
?>

File: POOPlatzi/JAVA/uberPool.java
public class uberPool extends Car {

String marca;
String modelo;


public uberPool(Integer id, String licencia, Account conductor, String pasajero, String marca,String modelo){

    super(id,licencia, conductor,pasajero);
    
    this.marca= marca;
    this.modelo= modelo;

}
}
